src/utils/authentication.ts

```diff
@@ -27,7 +27,14 @@ const authenticateJWT = (req: Request, res: Response, next: NextFunction) => {
                 req.user = null
             }
             next();
-        } catch (e) {
+        } catch (e: any) {
```

Comment:
I don't think the error should be typed as any here

Reply:
I agree, but apparently a catch block only allows any or unknown typing (or at least that's what my VS code linter said). Maybe there is a way to expect an Error type without a catch block?

Reply:
Fixed it.